lesson8/src/layouts/Layout.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react';
+import Menu from '../components/Menu';
+import MenuItem from '../components/MenuItem';
+
+const Layout = props => {
+    const [brand, setBrand] = useState('React blog!');
```

Comment:
```suggestion
    const brand = 'React blog!';
```
мы не собираемся изменять название брэнда)